src/antd-components/TimePicker.tsx

```
@@ -0,0 +1,33 @@
+import React from "react";
+import type { TimePickerProps } from "antd";
+import { Space, TimePicker } from "antd";
+import TreeSelect from "./TreeSelect";
+
+const onChange = (time, timeString) => {
+  console.log(time, timeString);
+};
+
+const TimePicker = () => (
```

Biome

[error] Shouldn't redeclare 'TimePicker'. Consider to delete it or rename it.

'TimePicker' is defined here:


(lint/suspicious/noRedeclare)